doc/source/conf.py

```diff
@@ -172,9 +172,9 @@ def setup(app):
 # built documents.
 #
 # The short X.Y version.
-version = 'beta'
+version = '0.4'
```

Comment:
Comments are misleading, this should be `0.4.0` after all, and `release` would most of the time be identical.